bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/embeddable/package-info.java

```diff
@@ -13,7 +13,7 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-@Version("1.1.1")
+@Version("2.0.0")
```

Comment:
Is this major bump really needed? It's a breaking change and we should avoid it.